ci/jenkins/jobs/macros.yaml

```diff
@@ -69,6 +69,21 @@
 
           if !(test -f jenkins/SECRET_EXIST); then
             echo "=== Fail to get secret ==="
+            
+            echo "=== Remove too many logs ==="
+            export LOG_DIR=/var/lib/jenkins/antrea_logs
+            find ${LOG_DIR}/* -type d -mmin +10080 | xargs rm -rf
+
+            echo "=== Saving capi logs ==="
+            mkdir -p ${LOG_DIR}/${cluster}/capi
+            capi_pod=$(kubectl get -n capi-system pods -o name | cut -d'/' -f 2)
+            kubectl logs ${capi_pod} -n capi-system --tail=100 > ${LOG_DIR}/${cluster}/capi/${capi_pod}
```

Comment:
does that work for multiple Pods? Or is there a guarantee that there is a single Pod in the `capi-system` namespace?

Reply:
Yes, only one pod.

Reply:
So far I only see one pod there but have a loop for guarantee is better. Thanks.